stories/user-actions-menu/user-actions-menu.stories.js

```diff
@@ -0,0 +1,742 @@
+export default {
+    title: 'Components/User Actions Menu',
+    parameters: {
+        description: `The user actions menu can be used as a standalone component, linked to another component, or closely connected to the ShellBar.<br>
+            The user actions menu displays an avatar button that triggers a Popover.<br>
+            The user actions menu can have multi-level navigation.
+        `,
+        tags: ['f3', 'theme'],
+        components: ['user-actions-menu', 'shellbar', 'popover', 'list', 'input', 'avatar', 'bar', 'button', 'icon', 'title', 'badge']
+    }
+};
+
+export const shellbar = () => `<div style="height: 400px;">
+    <div class="fd-shellbar">
+        <div class="fd-shellbar__group fd-shellbar__group--product">
+            <span class="fd-shellbar__logo">
+                <img src="//unpkg.com/fundamental-styles/dist/images/sap-logo.png" srcset="//unpkg.com/fundamental-styles/dist/images/sap-logo@2x.png 1x, //unpkg.com/fundamental-styles/dist/images/sap-logo@3x.png 2x, //unpkg.com/fundamental-styles/dist/images/sap-logo@4x.png 3x" width="48" height="24" alt="SAP">
+            </span>
+            <div class="fd-shellbar__group fd-shellbar__group--actions">
+                <div class="fd-shellbar__action">
+                    <div class="fd-popover fd-popover--right fd-user-actions-menu">
+                        <div class="fd-popover__control">
+                            <span
+                                class="fd-avatar fd-avatar--xs fd-avatar--circle fd-avatar--thumbnail"
+                                aria-controls="popoverA1"
+                                aria-expanded="true"
+                                aria-haspopup="true"
+                                aria-label="Avatar"
+                                style="background-image: url('assets/images/avatars/3.svg');"
+                                onclick="onPopoverClick('popoverA1');"
+                                role="button"
+                                tabindex="0"></span>
+                        </div>
+                
+                        <div class="fd-popover__body fd-popover__body--right" aria-hidden="false" id="popoverA1" style="width: 18rem !important;">
+                            <div class="fd-user-actions-menu__body fd-user-actions-menu__body--padding">
```

Comment:
You can't have this: `style="width: 18rem !important;"`
The 18rem width is part of the User Menu specs and needs to be in its css